config/src/unix.rs

```diff
@@ -48,6 +48,10 @@ pub struct UnixDomain {
 
     #[serde(default = "default_write_timeout")]
     pub write_timeout: Duration,
+
+    /// Don't use default_local_echo_threshold_ms() here to
+    /// disable the predictive echo for Unix domains by default.
+    pub local_echo_threshold_ms: Option<u64>,
```

Comment:
as mentioned below, this option doesn't apply to unix domains, so we shouldn't add it here